js/packages/core-base/src/check-and-close.ts

```diff
@@ -18,6 +18,20 @@ export function makeCheckAndClose({requests, logger: defaultLogger}: Options) {
     settings?: CheckSettings & CloseSettings
     logger?: Logger
   }): Promise<TestResult[]> {
+    if (!settings) settings = {}
+    settings.normalization = {
+      ...settings.normalization,
+      limit: {
+        maxImageHeight: Math.min(
+          settings.normalization?.limit?.maxImageHeight ?? Infinity,
+          requests.test.account.maxImageHeight,
+        ),
+        maxImageArea: Math.min(
+          settings.normalization?.limit?.maxImageArea ?? Infinity,
+          requests.test.account.maxImageArea,
+        ),
+      },
+    }
```

Comment:
better use a new nullish coalescing operator
```suggestion
    settings ??= {}
    settings.normalization ??= {}
    settings.normalization.limit ??= {}
    settings.normalization.limit.maxImageHeight = Math.min(
      settings.normalization.limit.maxImageHeight ?? Infinity,
      requests.test.account.maxImageHeight,
    )
    settings.normalization.limit.maxImageArea = Math.min(
      settings.normalization.limit.maxImageArea ?? Infinity,
      requests.test.account.maxImageArea,
    )
```